### DISCUSSION TITLE
Why repository was moved out from react-native-community?

### DISCUSSION THREAD

**Discussion by FRizzonelli** [react-native-async-storage/async-storage#507] [Q&A] [answered]
Hello guys!

Just a genuine question here. I'd like to know why repository was moved again. Back in the days we're used to install it directly from react-native and, fortunately, RN team decide to move it out from the core which is a super good thing! 
It was moved to @react-native-community organization which I believe is a good place to keep repos and materials and has a lot of collaborators.

I'd like to know what is the reason of this change, especially because I think it'll be complicated and confusing for new developers to keep track of this new change.

Thanks for any kind of response and feel free to close if this isn't the right place to ask! Also thank you for all the effort you put in open source software ❤️

**Comment by tido64** [comment, marked as answer]
Please take a look at [What repositories belong in React Native Community?](https://github.com/react-native-community/discussions-and-proposals/blob/master/partners/0001-organization-repository-policy.md), and the discussion here: https://github.com/react-native-community/discussions-and-proposals/issues/176#issuecomment-693569246

**Comment by pke** [comment]
It still confusing that the community version is at 1.12.1 and the non-community version at 1.14.1 according to VSCode hints.

**Reply by krizzu** [reply]
The community version is deprecated on npm, so you should go with react-native-async-storage org

**Reply by pke** [reply]
I found that out the hard way ;) Yarn didn't warn about it

**Reply by jarredt** [reply]
+1 to this -- had no idea the community version had been deprecated, so dependabot never let us know about the last several versions!

**Reply by krizzu** [reply]
I've deprecated the package long time ago, wonder if yarn/npm is not reporting this?
https://www.npmjs.com/package/@react-native-community/async-storage
